addons/l10n_ar/data/account.account.template.csv

```diff
@@ -1,261 +1,261 @@
-l10n_ar.base_retencion_iibb_lp_sufrida,l10n_ar.l10nar_base_chart_template,1.1.4.02.620,asset_current,Retención IIBB La Pampa sufrida,False
-l10n_ar.base_percepcion_iibb_lp_sufrida,l10n_ar.l10nar_base_chart_template,1.1.4.02.630,asset_current,Percepción IIBB La Pampa sufrida,False
-l10n_ar.base_saldo_favor_iibb_rn,l10n_ar.l10nar_base_chart_template,1.1.4.02.640,asset_current,Saldo a favor IIBB Río Negro,False
-l10n_ar.base_retencion_iibb_rn_sufrida,l10n_ar.l10nar_base_chart_template,1.1.4.02.650,asset_current,Retención IIBB Río Negro sufrida,False
-l10n_ar.base_percepcion_iibb_rn_sufrida,l10n_ar.l10nar_base_chart_template,1.1.4.02.660,asset_current,Percepción IIBB Río Negro sufrida,False
-l10n_ar.base_saldo_favor_iibb_az,l10n_ar.l10nar_base_chart_template,1.1.4.02.670,asset_current,Saldo a favor IIBB Santa Cruz,False
-l10n_ar.base_retencion_iibb_az_sufrida,l10n_ar.l10nar_base_chart_template,1.1.4.02.680,asset_current,Retención IIBB Santa Cruz sufrida,False
-l10n_ar.base_percepcion_iibb_az_sufrida,l10n_ar.l10nar_base_chart_template,1.1.4.02.690,asset_current,Percepción IIBB Santa Cruz sufrida,False
-l10n_ar.base_saldo_favor_iibb_tf,l10n_ar.l10nar_base_chart_template,1.1.4.02.700,asset_current,Saldo a favor IIBB Tierra del Fuego,False
-l10n_ar.base_retencion_iibb_tf_sufrida,l10n_ar.l10nar_base_chart_template,1.1.4.02.710,asset_current,Retención IIBB Tierra del Fuego sufrida,False
-l10n_ar.base_percepcion_iibb_tf_sufrida,l10n_ar.l10nar_base_chart_template,1.1.4.02.720,asset_current,Percepción IIBB Tierra del Fuego sufrida,False
+l10n_ar.base_cheques_de_terceros,l10n_ar.l10nar_base_chart_template,1.1.1.02.010,asset_cash,Third Party Checks,False
+l10n_ar.base_cheques_de_terceros_rechazados,l10n_ar.l10nar_base_chart_template,1.1.1.02.020,asset_current,Rejected Third Party Checks,False
+l10n_ar.base_fondo_comun_de_inversion,l10n_ar.l10nar_base_chart_template,1.1.2.01.010,asset_current,Mutual fund,False
+l10n_ar.base_plazo_fijo,l10n_ar.l10nar_base_chart_template,1.1.2.01.020,asset_current,Fixed term,False
+l10n_ar.base_deudores_por_ventas,l10n_ar.l10nar_base_chart_template,1.1.3.01.010,asset_receivable,Sales receivables,True
+l10n_ar.base_deudores_por_ventas_pos,l10n_ar.l10nar_base_chart_template,1.1.3.01.020,asset_receivable,Sales receivables (PoS),True
+l10n_ar.base_ret_percepcion_tasa_municipal,l10n_ar.l10nar_base_chart_template,1.1.4.01.010,asset_current,Ret/Perception Municipal Tax,False
+l10n_ar.base_saldo_a_favor_tasa_municipal,l10n_ar.l10nar_base_chart_template,1.1.4.01.020,asset_current,Balance in favor Municipal Tax,False
+l10n_ar.base_saldo_favor_iibb_caba,l10n_ar.l10nar_base_chart_template,1.1.4.02.010,asset_current,Balance in favor IIBB CABA,False
+l10n_ar.base_retencion_iibb_caba_sufrida,l10n_ar.l10nar_base_chart_template,1.1.4.02.020,asset_current,IIBB CABA withholding suffered,False
+l10n_ar.base_percepcion_iibb_caba_sufrida,l10n_ar.l10nar_base_chart_template,1.1.4.02.030,asset_current,Perception of IIBB CABA suffered,False
```

Comment:
I'm not sure, but wouldn't "incurred" be better here than "suffered"? Maybe we could quickly check with a Benj (He knows a bit of Spanish too)